pages/api/token/temporary.ts

```diff
@@ -143,4 +160,114 @@ const sendTemporaryTokenByEmail = async (email: string, temporaryToken: string)
     });
 };
 
-export default middleware([cors({ allowedMethods: ["POST"] }), validBearerToken()], handler);
+const requestContainsEmail = (): MiddlewareRequest => {
+  return async (req: NextApiRequest, res: NextApiResponse): Promise<MiddlewareReturn> => {
+    const email = req.body["email"];
+
+    if (!email) {
+      res.status(400).json({ error: "Invalid payload" });
+      return { next: false };
+    }
+
+    return { next: true, props: { email } };
+  };
+};
+
+const canLogIn = (): MiddlewareRequest => {
```

Comment:
This middleware function should probably be moved to it's own middleware file.  We can call it something like 'lockedOut.ts`?

Reply:
I guess it will depend on our little discussion in your first comment on this PR but I feel like middleware functions being used in one specific place the code should not be placed outside of that place.